deploy/docker/docker-entrypoint.sh

```diff
@@ -116,7 +126,7 @@ do
         VAR_FULL_NAME=$(echo "$VAR" | sed -r "s/([^=]*)=.*/\1/g")
         # Config in emq.conf
         if [[ ! -z "$(cat $CONFIG |grep -E "^(^|^#*|^#*\s*)$VAR_NAME")" ]]; then
-            echo "$VAR_NAME=$(eval echo \$$VAR_FULL_NAME)"
+            echo "$(echo_value)"
```

Comment:
I think it is enough to call `echo_value` function directly on this line, because `echo` is already executed on line 114, there is no need to `echo` again

Reply:
resolved in 6bd5ed4f145e957bf35d3b9f567e1a3600224407